src/services/goals.js

```diff
@@ -1315,7 +1315,7 @@ async function createObjectivesForGoal(goal, objectives, report) {
         savedObjective = await Objective.create({
           ...updatedObjective,
           title: objectiveTitle,
-          status: OBJECTIVE_STATUS.NOT_STARTED, // Only the hook should set status.
+          status: status || OBJECTIVE_STATUS.NOT_STARTED,
```

Comment:
Lets go over this tomorrow. 

This change was made to prevent a duping objective issue because the status is the ARO status. I think anytime we create a Objective for the first time 'Create' it should be 'Not Started'.

Reply:
ok sure, it is creating a different duping objective issue so we'll have to sort that out